README.md

```diff
@@ -1,52 +1,64 @@
-# Codius Manifest Format
+# Codius Manifest
 
 [Codius](https://codius.org) is an open-source decentralized hosting platform using [Interledger](https://interledger.org). It allows anyone to run software on servers all over the world and pay using any currency. Users package their software inside of [containers](https://www.docker.com/what-container). Multiple containers can run together inside of a [pod](https://kubernetes.io/docs/concepts/workloads/pods/pod/).
 
-**Codius Manifest** (this repository) is a module for validating and configuring Codius manifests.
+**Codius Manifest** (this repository) is a module for validating and generating Codius manifests.
 The Codius manifest format allows users to specify container images, public and
 private environment variables, and other information about pods. Manifests are used
 by [Codius hosts](https://github.com/codius/codiusd) to setup the container environments and download images.
 
-## Manifest Example
-Manifests must match the standard format, which is specified [here](https://github.com/codius/manifest/blob/master/src/schemas/ManifestSpec.json).
+## Manifest Format
+Manifests must match the standard format, which is specified [here](https://github.com/codius/manifest/blob/master/src/schemas/GeneratedManifestSpec.json).
+Manifests that are valid against the standard schema are considered complete.
 ```json
 {
-  "name": "test-app",
-  "version": "1.0.0",
-  "machine": "small",
-  "port": "8080",
-  "containers": [{
-    "id": "app",
-    "image": "hello-world@sha256:f5233545e43561214ca4891fd1157e1c3c563316ed8e237750d59bde73361e77",
-    "command": ["/bin/sh"],
-    "workdir": "/root",
-    "environment": {
-      "AWS_ACCESS_KEY": "$AWS_ACCESS_KEY",
-      "AWS_SECRET_KEY": "$AWS_SECRET_KEY"
+  "manifest": {
```

Comment:
should manifest be here? The hash is on the portion covered by the brackets. 
@sharafian  curious on your thoughts.